schema.sql

```diff
@@ -19,12 +19,36 @@ CREATE TABLE species (
   id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
   name VARCHAR(100) NOT NULL );
 
+CREATE TABLE vets (
+  id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
+  age INT,
+  date_of_graduation DATE );
+
+CREATE TABLE specializations (
+  sid INT REFERENCES species(id),
+  vid INT REFERENCES vets(id),
```

Comment:
[OPTIONAL ]
- `sid` and `vid` are not good column names. I sugges you use something like `species_id` and `vet_id`. They make them easily read and identified when the database grows.